pcsx2/DebugTools/SymbolMap.h

```diff
@@ -19,6 +19,7 @@
 #include <set>
 #include <map>
 #include <string>
+#include <mutex>
 
 #include "Utilities/Threading.h"
```

Comment:
I updated the commit to remove the line. I briefly tested it on Linux, but it shouldn't affect anything.